extensions/wikia/SpecialHealthcheck/SpecialHealthcheck_body.php

```diff
@@ -60,7 +60,7 @@ public function execute( $par ) {
 
 
 		$maxLoad = $wgRequest->getVal('maxload');
-		$cpuCount = rtrim( shell_exec('cat /proc/cpuinfo | grep processor | wc -l') );
+		$cpuCount = wfShellExec('cat /proc/cpuinfo | grep processor | wc -l');
```

Comment:
4 processes are needlesly spawned here. I would suggest reading and parsing cpuinfo directly